reference/fleet/logstash-output.md

```diff
@@ -41,7 +41,7 @@ input {
     ssl_certificate_authorities => ["<ca_path>"]
```

Comment:
GH UI doesn't allow me suggest, let's replace `ssl` with `ssl_enabled`.

Reply:
Ahhhh, Line 40, I'm guessing.  I'll handle it!